modules/ossm-config-kiali-otel.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/kiali/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-kiali-otel_{context}"]
+= Configuring Kiali with {DTProductName}
+
+When Kiali is properly configured with {DTProductName}, you can view distributed traces in the Kiali console. These traces provide insight into the communication between services within the service mesh, helping you understand how requests are flowing through your system and where potential issues may lie.
+
+The {DTShortName} integration allows Kiali to display trace overlays and details on the graph, scatterplot charts and in-depth trace/span information on detail pages, and integrated span information in logs and metric charts. Kiali also offers links to the external tracing UI.
+
+.Prerequisites
+
+* {SMProductName} is installed.
+
+* You have configured {DTShortName} with {SMProductName}.
+
+.Procedure
+
+. Setup Kiali to access traces from the {TempoName} frontend:
++
+.Example Tempo frontend
+[source,yaml]
+----
+external_services:
+  grafana:
+    enabled: true #<1>
+    url: "http://grafana-istio-system.apps-crc.testing/"
+  tracing:
+    enabled: true
+    provider: tempo
+    use_grpc: false
+    in_cluster_url: http://tempo-sample-query-frontend.tempo:3200
+    url: 'https://tempo-sample-query-frontend-tempo.apps-crc.testing'
+    tempo_config: #<2>
+      org_id: "1"
+      datasource_uid: "a8d2ef1c-d31c-4de5-a90b-e7bc5252cd00"
+----
+<1> Needed to see the "View in Tracing" link from the **Traces** tab.
+<2> Needed to see the "View in Tracing" link from the **Traces** tab and redirect to the proper Tempo `datasource`.
```

Comment:
```suggestion
<2> Required so that you can see the "View in Tracing" link from the **Traces** tab and redirect to the proper Tempo `datasource`.
```